### ISSUE TITLE
GCop656 false positive

### ISSUE THREAD

**Issue by hamidmayeli** [Geeksltd/GCop#312]
![image](https://user-images.githubusercontent.com/33150399/80393060-e1f3f900-88a7-11ea-9cd1-6f5c400d9c7e.png)

GCop suggest to change this code to 
```csharp
Order = order ?? "010";
```
which is a compile error.
